Britannia/common/buildings/BTWK_special_buildings.txt

```diff
@@ -266,26 +265,29 @@ btwk_aquae_sulis_baths_01 = {  # Disrepair
 }
 
 btwk_aquae_sulis_baths_02 = {  # Restored
-	construction_time = slow_construction_time
-	type_icon = "icon_structure_colosseum.dds"
+	construction_time = very_slow_construction_time
+	type_icon = "icon_structure_palace_of_achen.dds"
+	# Add feast cost reduction effect - weaker than leisure palaces 
 
 	can_construct = {
 		culture = { 
 			has_innovation = innovation_manorialism
-			culture_is_imperial_like_culture_trigger = yes
 		}
 	}
 
-	cost_gold = expensive_building_tier_5_cost
+	cost_gold = expensive_building_tier_6_cost  # Bit cheaper than full leisure palaces upgrade, bit weaker but other bonuses
 
 	county_modifier = {
 		development_growth_factor = 0.1
-		popular_opinion = 15
+		tax_mult = 0.2
+		county_opinion_add = 20
 	}
 
 	character_modifier = {
-		monthly_prestige = 0.5
-		stress_loss_mult = 0.1
+		monthly_prestige_gain_mult = 0.15
+		stress_loss_mult = 0.25
```

Comment:
Might be a bit strong?

Reply:
Here's the level 3 leisure palaces which I partly based the changes on:

```

character_modifier = {
		monthly_prestige_gain_mult = 0.15
		stress_loss_mult = 0.35
		owned_hostile_scheme_success_chance_add = 25
		owned_personal_scheme_success_chance_add = 25
	}
	
	duchy_capital_county_modifier = {
		monthly_county_control_change_factor = 0.6
		monthly_county_control_change_add = 0.4
	}	
```
It already has lower stress loss mult, but I reckon I could probably reduce both by 5%?

Reply:
Oh lol I didn't realize that vanilla had such a high chance... I guess 25% is probably fine then, up to you